app/authentication/login/login.component.e2e-spec.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->